src/Combinatorics/Matroids/matroids.jl

```diff
@@ -742,13 +742,38 @@ Construct the Pappus matroid.
 """
 pappus_matroid() = Matroid(Polymake.matroid.pappus_matroid())
 
+
+"""
+    moebius_kantor_matroid() 
+
+Construct the Möbius-Kantor matroid.
+"""
+moebius_kantor_matroid() = matroid_from_revlex_basis_encoding("0******0******0**********0********0*******0****0**0*****", 3, 8)
+
+"""
+    perles_matroid() 
+
+Construct the Perles matroid.
+"""
+
+perles_matroid() = matroid_from_revlex_basis_encoding("0000************0**********0****0**********0****0****************0*****0******0*****", 3, 9)
+
+
 """
     vamos_matroid()
 
 Construct the Vamos matroid.
 """
 vamos_matroid() = Matroid(Polymake.matroid.vamos_matroid())
 
+"""
+    R10_matroid() 
+
+Construct the R10 matroid.
+"""
+
```

Comment:
```suggestion
```
Again. Have you tried building the docs? https://docs.oscar-system.org/dev/DeveloperDocumentation/documentation/#build_doc